R/data.R

```diff
@@ -0,0 +1,64 @@
+#' Random adverse events
+#'
+#' @description Random adverse events
+#' @docType data
+#'
+#' @usage data(rADAE)
```

Comment:
```suggestion
#' @usage teal.modules.general::rADAE
```

Hey, checkout Warning section at the end of this chapter to see why we should avoid data() these days. https://r-pkgs.org/data.html#sec-data-data Can you also put package name in from of the dataset, so it's clear this dataset is from this package? I found rADAE in other packages as well, like teal.transform. 

Can you apply this comment to all packages in this file?

Reply:
Done! https://github.com/insightsengineering/teal.modules.general/pull/534/commits/8317d53b0b1d7971eed34f51c03296677eb4746a